aten/src/ATen/FunctionalStorageImpl.h

```diff
@@ -31,14 +31,16 @@ struct ViewMeta {
   ViewMeta(
       std::function<Tensor(const Tensor&, int64_t)> forward,
       std::function<Tensor(const Tensor&, const Tensor&, int64_t)> reverse,
+      bool has_symbolic_inputs,
```

Comment:
So what, this should be true if the std::function closes over SymInts, or something? Spell it out.